change/@fluentui-react-focus-2020-11-19-18-31-00-focusZoneConsumeTheme7.0.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```

Comment:
I wonder if this should be minor since it's adding a dependency? Not sure we have a standard policy about that.

Reply:
Yeah, I wondered what should happen here, my take so far is that if no API files change then it's patch, but I could make it minor if need be.